packages/react-router-redux/modules/__tests__/ConnectedRouter-test.js

```diff
@@ -95,4 +97,31 @@ describe('A <ConnectedRouter>', () => {
 
     expect(store.getState()).toHaveProperty('router.location.pathname', '/test')
   })
+
+  it('stays in sync if other middlewares dispatch routerActions as a reaction to the inital LOCATION_CHANGE', () => {
```

Comment:
I'm aware that this is an integration-test, rather than a unit-test... I hope that's ok, though. Maybe it should be placed somewhere else?

Reply:
I'm fine with full e2e testing, as long as we've got coverage.